### PR TITLE
vimPlugins: add bullets-vim

### PR DESCRIPTION
**Pull request by ajaxbits**
###### Motivation for this change
Add the [bullets.vim](https://github.com/dkarter/bullets.vim) plugin.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by teto** [comment]
drop the generic update commit to ease the merge

**Comment by teto** [comment]
you are taking the hard road, just run `git rebase -i master` and you drop the commits (replace the "pick" with "drop" or "d")

**Comment by teto** [comment]
please pull your master branch and rebase. Your local HEAD must be quite old: pkgs/misc/vim-plugins/vim-plugin-names is now in csv format (also it's now in pkgs/applications/editors/vim/plugins/vim-plugin-names).

**Comment by ajaxbits** [comment]
ty @teto -- 2f43659af1669a99d20920fa75c98e8e6cd44ff6 uses the newer CSV structure

File: pkgs/applications/editors/vim/plugins/vim-plugin-names [hunk 1]
```diff
@@ -74,6 +74,7 @@ https://github.com/fruit-in/brainfuck-vim/,,
 https://github.com/famiu/bufdelete.nvim/,,
 https://github.com/jlanzarotta/bufexplorer/,,
 https://github.com/akinsho/bufferline.nvim/,,
+https://github.com/dkarter/bullets.vim/,,bullets-vim
```
[the remark this answers is not in the document]
Comment (ajaxbits) [reply pkgs/applications/editors/vim/plugins/vim-plugin-names]: addressed in fd179dc1bd6a989a809e411966e9d432d2c67cf1

**Review by teto — APPROVED** [review]

**Pull request merged**